src/coreclr/vm/jitinterface.cpp

```diff
@@ -15019,6 +15019,23 @@ void EECodeInfo::Init(PCODE codeAddress, ExecutionManager::ScanFlag scanFlag)
     if (pRS == NULL)
         goto Invalid;
 
+#ifdef FEATURE_INTERPRETER
+    if (pRS != NULL && pRS->_flags & RangeSection::RANGE_SECTION_RANGELIST)
+    {
+        if (pRS->_pRangeList->GetCodeBlockKind() == STUB_CODE_BLOCK_STUBPRECODE)
+        {
+            PTR_StubPrecode pStubPrecode = dac_cast<PTR_StubPrecode>(PCODEToPINSTR(codeAddress));
+            if (pStubPrecode->GetType() == PRECODE_INTERPRETER)
+            {
+                codeAddress = dac_cast<PTR_InterpreterPrecode>(pStubPrecode)->GetData()->ByteCodeAddr;
```

Comment:
This will end up creating codeinfo for a different IP than the one passed into Init. I can imagine that it is fixing a problem somewhere, but it is likely creating different problems somewhere else. Is this going to break places like https://github.com/dotnet/runtime/blob/2a958314361c0b773caa681358835135f8f139e6/src/coreclr/vm/threadsuspend.cpp#L5787-L5789 where we expect the valid EECodeInfo to have real code with GCInfo, etc. behind it?